examples/getting_started/pt/nvflare_pt_getting_started.ipynb

```diff
@@ -0,0 +1,412 @@
+   "id": "13771bfb-901f-485a-9a23-84db1ccd5fe4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nvflare import FedAvg, FedJob, ScriptExecutor\n",
+    "\n",
+    "job = FedJob(name=\"cifar10_fedavg\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9361d9f8-54f3-4363-b3ba-706a7ae3a8e9",
+   "metadata": {},
+   "source": [
+    "#### 3. Define the Controller Workflow\n",
+    "Define the controller workflow and send to server."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6962e6cc-995e-4356-8156-3ceba2c7a249",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n_clients = 2\n",
+    "\n",
+    "controller = FedAvg(\n",
+    "    min_clients=n_clients,\n",
+    "    num_rounds=2,\n",
+    ")\n",
+    "job.to(controller, \"server\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a63ce0c-ad3e-4434-b2a8-c8f2a4c2e7a5",
+   "metadata": {},
+   "source": [
+    "#### 4. Create Global Model\n",
+    "Now, we create the initial global model and send to server."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0e2c514c-7758-4d30-bb5c-ae3c63be50aa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from src.net import Net\n",
+    "\n",
+    "job.to(Net(), \"server\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77f5bc7f-4fb4-46e9-8f02-5e7245d95070",
+   "metadata": {},
+   "source": [
+    "That completes the components that need to be defined on the server."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "548966c2-90bf-47ad-91d2-5c6c22c3c4f0",
+   "metadata": {},
+   "source": [
+    "#### 5. Add clients\n",
+    "Next, we can use the `ScriptExecutor` and send it to each of the clients to run our training script.\n",
+    "\n",
+    "Note that our script could have additional input arguments, such as batch size or data path, but we don't use them here for simplicity.\n",
+    "We can also specify, which GPU should be used to run this client, which is helpful for simulated environments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ad5d36fe-9ae5-43c3-80bc-2cdc66bf7a7e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in range(n_clients):\n",
+    "    executor = ScriptExecutor(\n",
+    "        task_script_path=\"src/cifar10_fl.py\", task_script_args=\"\"  # f\"--batch_size 32 --data_path /tmp/data/site-{i}\"\n",
+    "    )\n",
+    "    job.to(executor, f\"site-{i}\", gpu=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "113fd6af-85be-4f75-8a8e-4666771252b3",
+   "metadata": {},
+   "source": [
+    "That's it!\n",
+    "\n",
+    "#### 6. Optionally export the job\n",
+    "Now, we could export the job and submit it to a real NVFlare deployment using the [Admin client](https://nvflare.readthedocs.io/en/main/real_world_fl/operation.html) or [FLARE API](https://nvflare.readthedocs.io/en/main/real_world_fl/flare_api.html). "
```

Comment:
Maybe we put this part after the simulator running?
Because it's for the real world?